### PR TITLE
Replace crypto.randomBytes() with Math.random() to address #563

### PR DESCRIPTION
**Pull request by duzun**
`Math.random()` is of lower quality then `crypto.randomBytes()`, but is much faster and doesn't consume system's entropy.

### CODE REVIEW

File: src/command.js [hunk 1]
```diff
@@ -13,10 +11,25 @@ export default class Command {
...
 }
+
+/**
+ * Generate an ascii random ID
+ * 
+ * @param  {Number} bytes number of bytes the ID should contain
+ * @return {String} a textual ID of `bytes` entropy
+ */
+function randId(bytes) {
+    var ret = [];
```
Comment (amir20) [inline src/command.js]: can you convert this to `const`?

File: src/command.js [hunk 2]
```diff
@@ -13,10 +11,25 @@ export default class Command {
...
+    for( ; bytes > 0 ; bytes -= 4 ) {
+        // Make a unique string of pow(2, 32) entropy.
+        ret.push(((Math.random()*(-1>>>0))>>>0).toString(36)); // number in base 36 (to save space)
```
Comment (amir20) [inline src/command.js]: So `ret.push((Math.random() * (-1 >>> 0) >>> 0).toString(36));`? Notice the extra parenthesis.

File: src/out_object.js [hunk 3]
```diff
@@ -1,17 +1,31 @@
...
     }
 }
+
+/**
+ * Generate an ascii random ID
+ * 
+ * @param  {Number} bytes number of bytes the ID should contain
+ * @return {String} a textual ID of `bytes` entropy
+ */
+function randId(bytes) {
```
Comment (amir20) [inline src/out_object.js]: This again duplicated here? Feels more fitting if you create a file `util/random_id.js` and export it to be used here?

File: src/command.js [hunk 4]
```diff
@@ -13,10 +11,25 @@ export default class Command {
...
     }
 }
+
+/**
+ * Generate an ascii random ID
+ * 
+ * @param  {Number} bytes number of bytes the ID should contain
+ * @return {String} a textual ID of `bytes` entropy
+ */
+function randId(bytes) {
```
Comment (amir20) [inline src/command.js]: `bytes:number`

File: src/out_object.js (hunk 3 above)
Comment (duzun) [inline src/out_object.js]: I wasn't sure of coding style you use :-)

Comment (amir20) [inline src/out_object.js]: Oh I don't think that's code style. That's just removing redundant code.  Coding style is caught with flow and eslint.

**Comment by amir20** [comment]
Woohoo. Approving.

**Comment by amir20** [comment]
@duzun thanks! Do you need a new version now? I don't know if this was worth of a new npm push.

**Comment by duzun** [comment]
I would like to use the updated code in production.
Do you have other changes to do before a version bump?
This could be a minor version update.

**Comment by amir20** [comment]
Just pushed a new version. Cheers.

**Comment by duzun** [comment]
Tyvm. Cheers.

**Pull request merged**